### PR TITLE
Fix visualizations not showing graph name

### PR DESCRIPTION
**Pull request by cryptoquick**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #1084 

### The problem
Cell names weren't being passed to the visualization editor.

### The Solution
Pass the name property through to the Visualization component.

### CODE REVIEW

**Comment by jademcgough** [comment]
s-sugoi desu ne~

File: ui/src/dashboards/containers/DashboardPage.js [hunk 1]
```diff
@@ -205,12 +205,12 @@ const DashboardPage = React.createClass({
         }
         {
           isEditMode ?
-            <EditHeader
+            <DashboardHeaderEdit
               dashboard={dashboard}
               onCancel={this.handleCancelEditDashboard}
               onSave={this.handleRenameDashboard}
             /> :
-            <Header
```
Comment (jademcgough) [inline ui/src/dashboards/containers/DashboardPage.js]: this threw me off earlier, thank you for changing it to the actual component name

**Comment by jaredscheib** [comment]
@cryptoquick Update changelog before merging, otherwise good to go!

File: ui/src/dashboards/containers/DashboardPage.js (hunk 1 above)
Comment (cryptoquick) [inline ui/src/dashboards/containers/DashboardPage.js]: 😸

**Pull request merged**